io/net/Network_config.py

```diff
@@ -36,14 +37,17 @@ def setUp(self):
         '''
         To check and install dependencies for the test
         '''
-        sm = SoftwareManager()
-        for pkg in ["ethtool", "net-tools"]:
-            if not sm.check_installed(pkg) and not sm.install(pkg):
-                self.cancel("%s package is need to test" % pkg)
+        self.ipaddr = self.params.get("host_ip", default="")
+        self.netmask = self.params.get("netmask", default="")
         interfaces = netifaces.interfaces()
         self.iface = self.params.get("interface")
         if self.iface not in interfaces:
             self.cancel("%s interface is not available" % self.iface)
+        configure_network.set_ip(self.ipaddr, self.netmask, self.iface)
+        sm = SoftwareManager()
```

Comment:
Any reason to install packages after IP set ?

Reply:
yes i tried but i get error so i kept.